### DISCUSSION TITLE
UI-Shell token theme

### DISCUSSION THREAD

**Discussion by JemyDev** [carbon-design-system/carbon#7537] [Help] [answered]
Hello,

I'm actually implementing carbon design for a backoffice purpose, I added a switch theme to my website, but it doesn't change the color for my ui-shell components. Is there any PR incoming to fix this ? Or has it a trick to apply the theme to components ?

Thanks for your answer !

**Comment by joshblack** [comment, marked as answer]
Hi there, @JemyDev! 👋 

Unfortunately, the theme is not theme-able at the moment 😞  We're definitely hoping to introduce this as part of our work this year. When looking at it previously, we ran into some challenges with having it work in our theme system and any of the alternatives that we were looking at. Hopefully this year we can work through them and ship something to make it easy to use one of the shell themes 🤞 

Hope this helps!
